### PR TITLE
Dev: Azure Automation additional cmdlet bugfixes

### PR DESCRIPTION
**Pull request by safeermohammed**
Cmdlet bugfixes 
taking latest Automation SDK
Scenario tests

### CODE REVIEW

**Comment by hovsepm** [comment]
@safeermohammed remove restore.config file from the PR.

File: src/ResourceManager/Automation/Commands.Automation/packages.config [hunk 1]
```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Azure.Common" version="2.1.0" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Authentication" version="1.5.2-preview" targetFramework="net45" />
   <package id="Microsoft.Azure.Common.Dependencies" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Automation" version="0.50.2-prerelease" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Resources" version="2.18.11-preview" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Automation" version="2.0.0" targetFramework="net45" />
+  <package id="Microsoft.Azure.Management.Resources" version="2.18.7-preview" targetFramework="net45" />
```
Comment (hovsepm) [inline src/ResourceManager/Automation/Commands.Automation/packages.config]: do NOT downgrade any version. BTW, we are going to bump everyone to Resource 2.18.13.

Comment (safeermohammed) [inline src/ResourceManager/Automation/Commands.Automation/packages.config]: looks like incorrectly updated during merge within team, Fixed to be back to 2.18.11. 
Sure 2.18.13 should be fine for us

**Comment by hovsepm** [comment]
on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/451/

**Comment by safeermohammed** [comment]
Made exactly how it is in ASM scenario tests now.

**Pull request merged**